### DISCUSSION TITLE
Feature Request: Ability to maintain the JPG pictures even we ask to convert Media in MP4 format

### DISCUSSION THREAD

**Discussion by gionatanfazio** [sircharlo/meeting-media-manager#193] [Ideas] [answered]
Hello!
thanks for great app ..
Can you add this new feature?

Ability to maintain the JPG pictures even we ask to convert Media in MP4 format.
In this way all the Media would be available, for Zoom and VLC users.

thanks for great support
best regards, Gionatan
<img width="700" alt="2022-02-09_16-50-03" src="https://user-images.githubusercontent.com/38213936/153239006-4e5020a0-aeb7-47a6-a94e-23f0ed0ad974.png">

**Comment by sircharlo** [comment, marked as answer]
I like it, and therefore I've implemented it. 😆 
See https://github.com/sircharlo/jw-meeting-media-fetcher/commit/62bd499b86757a0330f52f9594e1e8210e18fc31. To be released in the upcoming days.
